torch/_torch_docs.py

```diff
@@ -3158,7 +3158,7 @@
 Indices are ordered from left to right according to when each was sampled
 (first samples are placed in first column).
 
-If :attr:`input` is a vector, :attr:`out` is a vector of size :math:`num_samples`.
+If :attr:`input` is a vector, :attr:`out` is a vector of size `num\_samples`.
 
```

Comment:
I think it should have been `:attr:` instead of math. Your fix won’t render properly either